pages/Admins/Admins.module.css

```diff
@@ -39,14 +26,79 @@
   border-radius: 7px;
   height: 33px;
   font-size: 14px;
-  float: right;
+  font-family: Avenir;
+  font-weight: 800;
+  align-self: center;
+  margin-left: auto;
 }
 
 .button:hover {
   background-color: #cecccc;
   color: #646464;
 }
 
-.buttons {
-  margin-left: 16%;
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 2%;
+  padding-right: 2%;
+  height: 10%;
+  border-bottom: 1px solid #ebebeb;
+  color: #646464;
+  font-size: 13px;
+}
+
+.header {
+  width: 90%;
+  display: flex;
+  align-items: flex-start;
+}
+
+.manageButton {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+
+  font-family: Avenir;
```

Comment:
You can remove this font for now! I will add the material UI global styling to fix this